app/pathogen/arbovirus/analyze/recharts/change-in-median-seroprevalence-over-time-graph.tsx

```diff
@@ -49,9 +52,13 @@ interface GenerateBucketsForTimeIntervalOutput {
 const generateBucketsForTimeInterval = (input: GenerateBucketsForTimeIntervalInput): GenerateBucketsForTimeIntervalOutput => {
   let currentIntervalStartDate = startOfYear(input.earliestDate);
   const buckets: TimeInterval[] = [];
-  
+
+  if(input.groupingBucketSize.years === 0) {
+    throw new Error("Unable to group data points with a bucket size of zero years.")
+  }
+
   while(isBefore(currentIntervalStartDate, input.latestDate)) {
-    const currentIntervalEndDate = endOfYear(addYears(currentIntervalStartDate, input.groupingBucketSize.years));
+    const currentIntervalEndDate = endOfYear(addYears(currentIntervalStartDate, input.groupingBucketSize.years - 1));
```

Comment:
Turns out I accidentally made a bucket size of 5 years equal 6 because of this algo. Whoops, fixed now. Only realized it when I tried to put 1 year into this lol